cloudformation/lib/s3.js

```diff
@@ -38,6 +38,49 @@ export default {
                 }
             }
         },
+        TemporaryAssetBucket: {
+            Type: 'AWS::S3::Bucket',
+            Properties: {
+                BucketName: cf.join('-', [cf.stackName, cf.accountId, cf.region, 'temp']),
+                PublicAccessBlockConfiguration: {
+                    RestrictPublicBuckets: true,
+                    IgnorePublicAcls: true,
+                    BlockPublicPolicy: true,
+                    BlockPublicAcls: true
+                },
+                VersioningConfiguration: {
+                    Status: 'Disabled'
+                },
+                RorsConfiguration: {
```

Comment:
I think we can omit the section on Cors as there is no public access to this bucket?